.github/workflows/publish.yml

```diff
@@ -29,7 +29,7 @@ jobs:
       - name: Publish
         run: |-
           sbt +publishSigned
-          CI_RELEASE=akka-grpc-codegen/publishSigned CI_SNAPSHOT_RELEASE=akka-grpc-codegen/publish sbt ++2.13.10\! publishSigned
+          CI_RELEASE=akka-grpc-codegen/publishSigned CI_SNAPSHOT_RELEASE=akka-grpc-codegen/publish
```

Comment:
I don't know the background for this, but could have been something with 2.12 and 2.13 cross publishing.
As far as I can see the above `sbt +publishSigned` will already publish for 2.13 and 3.

I don't know what that CI_RELEASE and CI_RELEASE is, but leaving that unchanged

Reply:
Looks like Raboof added that as a workaround for some publishing issues last year (https://github.com/akka/akka-grpc/issues/1400), not sure I understand why that is needed though.